nsqadmin/static/js/views/lookup.js

```diff
@@ -0,0 +1,77 @@
+var _ = require('underscore');
+var $ = require('jquery');
+
+var AppState = require('../app_state');
+var BaseView = require('./base');
+
+var Topic = require('../models/topic');
+var Channel = require('../models/channel');
+
+var LookupView = BaseView.extend({
+    className: 'lookup container-fluid',
+
+    template: require('./spinner.hbs'),
+
+    events: {
+        'click .hierarchy button': 'onCreateTopicChannel',
+        'click .delete-topic-link': 'onDeleteTopic',
+        'click .delete-channel-link': 'onDeleteChannel'
+    },
+
+    initialize: function() {
+        BaseView.prototype.initialize.apply(this, arguments);
+        $.ajax(AppState.url('/topics?inactive=true')).done(function(data) {
+            this.template = require('./lookup.hbs');
+            this.render({
+                'topics': _.map(data['topics'], function(v, k) {
+                    return {'name': k, 'channels': v};
+                })
+            });
+        }.bind(this));
+    },
+
+    onCreateTopicChannel: function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+        var topic = $(e.target.form.elements['topic']).val();
+        var channel = $(e.target.form.elements['channel']).val();
+        if (topic === '' && channel === '') {
+            return;
+        }
+        $.post(AppState.url('/topics'), JSON.stringify({
+            'topic': topic,
+            'channel': channel
+        })).done(function() {
```

Comment:
It would be great to surface errors from these calls; currently something like an invalid topic/channel name is just dropped here w/o user feedback.